test/blackbox-tests/test-cases/pkg/merlin/dev-tool-merlin-no-ocaml-lockfile.t

```diff
@@ -0,0 +1,20 @@
+Exercise the behaviour of "dune tools exec ocamlmerlin" when the lockdir
```

Comment:
I think we can skip this test. We are already testing it for other tools that require the compiler, and there aren't additional code paths handled to the logic here.